app/client/src/features/events/EventSidebar/EventSidebar.tsx

```diff
@@ -135,38 +133,39 @@ export const EventSidebar = ({ fragmentRef }: EventSidebarProps) => {
             >
                 <EventDetailsCard fragmentRef={data} />
                 <SpeakerList className={clsx(classes.item, classes.fullWidth)} fragmentRef={data} />
-                { // Moderator feedback state = 2, Participant feedback state = 1
-                    displayFeedbackButton ?
-                        <SubmitLiveFeedback 
+                {
+                    // Moderator feedback state = 2, Participant feedback state = 1
+                    displayFeedbackButton ? (
+                        <SubmitLiveFeedback
                             className={classes.fullWidth}
                             eventId={data.id}
                             // connectionKey='useLiveFeedbackListFragment_liveFeedback'
-                        /> :
+                        />
+                    ) : (
                         <AskQuestion
                             className={classes.fullWidth}
                             eventId={data.id}
                             connectionKey='useQuestionListFragment_questions'
-                    />
+                        />
+                    )
                 }
             </Grid>
 
             {!data.isViewerModerator && <QuestionCarousel fragmentRef={data} />}
 
-            { 
-                data.isViewerModerator &&
+            {data.isViewerModerator && (
                 <Tabs variant='scrollable' scrollButtons='auto' value={tabIndex} onChange={handleTabChange}>
                     <Tab label='Question Queue' />
                     <Tab label='Question List' />
                     <Tab label='Live Feedback' />
                 </Tabs>
-            }
-            { 
-                !data.isViewerModerator &&
+            )}
+            {!data.isViewerModerator && (
                 <Tabs value={tabIndex} onChange={handleTabChange}>
                     <Tab label='Question List' />
                     <Tab label='Live Feedback' />
                 </Tabs>
```

Comment:
With how many ternary's and conditionals there are, it may be worth splitting this up into two separate components.